eng/performance/benchmark_jobs.yml

```diff
@@ -0,0 +1,86 @@
+parameters:
+  name: ''             # required -- job name (prefix), should be [public|private]_osname
+  kind: ''             # required -- benchmark kind. As of today, only "micro" benchmarks are supported, we plan to add "scenarios" soon
+  architecture: ''     # required -- Architecture. Allowed values: x64, x86, arm32, arm64
+  pool: ''             # required -- name of the Helix pool
+  queue: ''            # required -- name of the Helix queue
+  container: ''        # optional -- id of the container
+  frameworks: []       # required -- list of frameworks. Allowed values: net461, netcoreapp2.0, netcoreapp2.1, netcoreapp2.2, netcoreapp3.0, corert
+
+jobs:
+- template: ../common/templates/jobs/jobs.yml
+  parameters:
+    enableTelemetry: false
+    enablePublishBuildArtifacts: true
+    helixRepo: dotnet/performance
+    jobs:
+      - job: sdk_performance_${{ parameters.name }}_${{ parameters.architecture }}
+        timeoutInMinutes: 320
+        variables:
+        - ${{ if eq(variables['System.TeamProject'], 'public') }}:
+          - name: BenchViewRunType
+            value: private
+          - name: BenchviewCommitName
+            value: '.NET Performance - $(BenchViewRunType) - $(Build.SourceBranchName) $(System.PullRequest.PullRequestNumber)'
+          # for public runs we want to generate data (--generate-benchview-data ) but don't upload it (missing --upload-to-benchview-container)
```

Comment:
We might want to separate out public + PR and public + not PR now so that when the new changes to the helix queues happen, we don't have to do it then. Also, once we move to the new visualization tool, we will likely allow anonymous upload so that the public jobs can push their data to the new benchview.